test/scripts/create_account.js

```diff
@@ -43,6 +43,12 @@ export async function createFundedAccount (roleID, balances) {
   }
 }
 
+export async function fundMasterAccount (asset, amount, assetOwner = assetHelper.masterKp) {
```

Comment:
there is no way to get master id in tests (except of hard coding or getting it from a helper so just in case dropped this helper here)